doc/interoperability/global-scope.md

```diff
@@ -202,7 +200,7 @@ The example means that it is not possible to dynamically look up a global variab
 
 ### Testing whether a global variable exists
 
-In Scala.js 0.6.x, it was possible to test whether a global variable exists (e.g., to perform a feature test) as follows:
+In earlier versions Scala.js, it was possible to test whether a global variable exists (e.g., to perform a feature test) as follows:
```

Comment:
There is the same problem here. IMO "Scala.js 0.6.x" remains better here.

As an alternative, we could write this a bit more with something like:
> In order to test whether a global variable exists (e.g., to perform a feature test), we might be tempted to use the following test:
> ...
> However, accessing `js.Dynamic.global.Promise` will throw a `ReferenceError` if `Promise` is not defined.
> Instead, you must use `js.typeOf`: